### PR TITLE
add #pragma once to header without it

### PR DESCRIPTION
**Pull request by tnixeu**
I noticed that some of the headers are missing `#pragma once`. This PR adds it to the remaining headers

### CODE REVIEW

File: src/OpenLoco/Map/QuarterTile.h [hunk 1]
```diff
@@ -1,3 +1,4 @@
+#pragma once
```
Comment (janisozaur) [inline src/OpenLoco/Map/QuarterTile.h:1]: Can you add one blank line after each pragma to match other headers?

File: resources/Resource.h [hunk 2]
```diff
@@ -1,3 +1,4 @@
+#pragma once
```
Comment (janisozaur) [inline resources/Resource.h:1]: this is a generated file, please remove this pragma

File: src/OpenLoco/Map/QuarterTile.h (hunk 1 above)
Comment (tnixeu) [reply src/OpenLoco/Map/QuarterTile.h:1]: I now have added the black line after the new pragmas

File: resources/Resource.h (hunk 2 above)
Comment (tnixeu) [reply resources/Resource.h]: The pragma is now removed from this file

**Review by duncanspumpkin — APPROVED** [review]

**Pull request merged**